jstests/core/dbadmin.js

```diff
@@ -10,6 +10,9 @@ t.save( { x : 1 } );
 var res = db._adminCommand( "listDatabases" );
 assert( res.databases && res.databases.length > 0 , "listDatabases 1 " + tojson(res) );
 
+var res = db.adminCommand( {listDatabases: 1, nameOnly: true} );
+assert( res.databases && res.databases.length > 0 && res.nameOnly , "listDatabases 1 " + tojson(res) );
```

Comment:
nit: The "listDatabases 1" message isn't very helpful, maybe do "listDatabases: 1, nameOnly: true" or just "listDatabases nameOnly".